nodes/index.adoc

```diff
@@ -17,6 +17,18 @@ Having stable and healthy nodes in a cluster is fundamental to the smooth functi
 In {product-title}, you can access, manage, and monitor a node through the `Node` object representing the node.
 Using the OpenShift CLI (`oc`) or the web console, you can perform the following operations on a node.
 
+The following components of a node are responsible for maintaining the running of pods and providing the Kubernetes runtime environment.
+
+* *Container runtime*: The container runtime is responsible for running containers. Kubernetes offers several runtimes such as containerd, cri-o, rktlet, and Docker.
+
+* *Kubelet*: Kubelet runs on nodes and reads the container manifests. It ensures that the defined containers have started and are running. The kubelet process maintains the state of work and the node server. Kubelet manages network rules and port forwarding. The kubelet manages containers that are created by Kubernetes only.
+
+* *Kube-proxy*: Kube-proxy runs on every node in the cluster and maintains the network traffic between the Kubernetes resources. A Kube-proxy ensures that the networking environment is isolated and accessible.
+
+* *DNS*: Cluster DNS is a DNS server which serves DNS records for Kubernetes services. Containers started by Kubernetes automatically include this DNS server in their DNS searches.
+
+image::295_OpenShift_Nodes_Overview_1222.png[Nodes overview]
```

Comment:
@xenolinux This alt text doesn't really describe what is depicted here. Do you want to revise or should I merge?

Reply:
Addressed